<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPairProjectClass</class>
 <widget class="QMainWindow" name="QtPairProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtPairProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>1261</width>
        <height>681</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1259</width>
         <height>839</height>
        </rect>
       </property>
       <widget class="QPushButton" name="changeRange">
        <property name="geometry">
         <rect>
          <x>1160</x>
          <y>30</y>
          <width>61</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>20</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 修改坐标范围</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>0</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>请注意，坐标不显示小数</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>40</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>请输入整数x，0&lt;x&lt;=100000</string>
        </property>
       </widget>
       <widget class="QPushButton" name="openFile">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>70</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>从文件导入</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>1060</x>
          <y>30</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="open">
        <property name="geometry">
         <rect>
          <x>1130</x>
          <y>70</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>手动导入</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>800</width>
          <height>800</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>120</y>
          <width>281</width>
          <height>581</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="paintPoint">
        <property name="geometry">
         <rect>
          <x>1130</x>
          <y>710</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>绘制交点</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>750</y>
          <width>281</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteItem">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>710</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>删除所选数据</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearAll">
        <property name="geometry">
         <rect>
          <x>1050</x>
          <y>710</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1286</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtPairProject.qrc"/>
 </resources>
 <connections/>
</ui>
